src/plugins/console/public/application/components/output_panel_empty_state.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { EuiHorizontalRule, EuiSpacer, EuiTitle, EuiLink, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useServicesContext } from '../contexts';
+
+export const OutputPanelEmptyState: FunctionComponent = () => {
+  const { docLinks } = useServicesContext();
+
+  return (
+    <div className="conApp__outputPanel__emptyState" data-test-subj="consoleOutputPanelEmptyState">
```

Comment:
Wasn't there a way to achieve this with just using EuiFlexGroup?

Reply:
Thanks for the suggestion @sabarasaba! I tried using EuiFlexGroup and it worked well; plus, it allows us to reduce the number of CSS properties needed.